### DISCUSSION TITLE
Welcome to fastlane-plugin-huawei_appgallery_connect Discussions!

### DISCUSSION THREAD

**Discussion by shr3jn** [shr3jn/fastlane-plugin-huawei_appgallery_connect#56] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Danial-haider** [comment]
Hi to upload the application using fastlane i had a problem on App Gallery i got this error
[31mfastlane finished with errors[0m [31m [!] {"ret":{"code":204144694,"msg":"[cfs] get siteId failed, additional msg is [[AppGalleryConnectFileService]distContryList is empty and usage route site is not China.]"}}[0m1
